packages/rum-core/src/domain/rumEventsCollection/error/errorCollection.spec.ts

```diff
@@ -131,6 +140,14 @@ describe('error collection', () => {
       })
     })
 
+    // it('should have feature flag of the current view', () => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏remove this commented test?